cmd/conduit/list.go

```diff
@@ -0,0 +1,173 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/spf13/cobra"
+
+	"github.com/algorand/indexer/exporters"
+	"github.com/algorand/indexer/importers"
+	"github.com/algorand/indexer/processors"
+)
+
+func makeListCmd() *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "list",
+		Short: "lists all plugins available to conduit",
+		Long:  "lists all plugins available to conduit",
+		Args:  cobra.MaximumNArgs(2),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runListCmd(os.Stdout, args)
+		},
+		SilenceUsage: true,
+		// Silence errors because our logger will catch and print any errors
+		SilenceErrors: true,
+	}
```

Comment:
We seem to be doing something wrong with the SilenceErrors / SilenceUsage / RunE configuration.

This command is interactive, output should be human readable. And we should print out the usage information when there's an error to help people use the command:

```
./conduit list importers
{"__type":"Conduit","_name":"main","level":"error","msg":"invalid argument, specify [importers, processors, exporters] before plugin name","time":"2022-11-03T10:22:58-04:00"}

./conduit list importers file_writer
{"__type":"Conduit","_name":"main","level":"error","msg":"no importer by name: file_writer","time":"2022-11-03T10:23:10-04:00"}


./conduit list exporter file_writer
{"__type":"Conduit","_name":"main","level":"error","msg":"invalid argument type (exporter), specify [importers, processors, exporters] before plugin name","time":"2022-11-03T10:23:15-04:00"}
```